src/PhpSpreadsheet/Reader/Html.php

```diff
@@ -878,14 +878,14 @@ private function applyInlineStyle(&$sheet, $row, $column, $attributeArray): void
 
                 case 'width':
                     $sheet->getColumnDimension($column)->setWidth(
-                        (float) str_replace('px', '', $styleValue)
+                        (float) str_replace(['px', 'pt'], '', $styleValue)
```

Comment:
I have half of a helper class that covers a wide range of the UoM that can be applied in html styles, with conversion factors for import to Spreadsheet styles... but this will work well enough for the moment... it's the relative units that are a real annoyance, because we can't adjudge `50%` accurately, as we don't know `50%` of what? I'll do that as a separate PR at some point though

Reply:
That would be really helpful, especially because Excel widths and heights aren't well documented. They aren't pixels, or points, or twips, or ... I know the default column width in Excel units and in pixels, but I haven't had time to establish whether there is a linear relationship between them. The Html writer and reader probably make things too wide or too narrow because of this.

Reply:
Didn't really mean non-linear above - it is linear, but it depends on the font. Which may be too sophisticated to try to emulate (as with your 50% comment). But I think it translates better to expressing the unit as ems rather than points or pixels.